<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="radio4d">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>train4dAll</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio6d">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>98</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>train6dAll</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Train">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Training</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Walk">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>560</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="knum">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>640</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>k:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>track4D.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>track6D.txt</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Track">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Track</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
